ccdb5_ui/templates/ccdb5_ui/ccdb-main.html

```diff
@@ -47,15 +47,43 @@
 {% endblock %}
 
 {% block page_js %}
-  <!--[if IE 9]><!-->
-  <script src="https://cdn.polyfill.io/v2/polyfill.js?features=Intl.~locale.en"></script>
-  <!--<![endif]-->
   <script src="{% static "ccdb5.min.js" %}" type="text/javascript"></script>
   <script type="text/javascript">
-    ReactDOM.render(
-      React.createElement(ccdb5_ui.App, {}, null),
-      document.getElementById('root')
-    );
+    function browserSupportsAllFeatures() {
```

Comment:
FYI: I am moving the bootstrapping code to `index.js` in this PR https://github.com/cfpb/ccdb5-ui/pull/106/files#diff-528c3923d718a8860f5d8c05c3931c55L18

Reply:
OK - I'll wait on that and then update. Thanks!